docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -86,7 +86,7 @@ The kindnet configuration file can be found in the [CAPI repo](https://raw.githu
 
```

Comment:
@kbreit Last one is in line.85 => https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/main/docs/book/src/tasks/experimental-features/yamls/kindnet-clusterresourceset.yaml 

Thank you very much! :)